Helpers/populateListHelper.jsx

```diff
@@ -0,0 +1,16 @@
+export function populateHTMLList(array) {
+    return array.map((arrayElement) =>
+        <li key={arrayElement.toString()}> {arrayElement.toString()}</li>
+    )
+}
+
+export function readyArrayForReactSelect(array) {
```

Comment:
I think this function would be better named something like `mapArrayToSelectOptions` which makes it clearer what is going on